contracts/swap/swap.go

```diff
@@ -76,7 +76,7 @@ func (s *Swap) ValidateCode(ctx context.Context, b bind.ContractBackend, address
 
 // Deploy a Swap contract
 func (s *Swap) Deploy(auth *bind.TransactOpts, backend bind.ContractBackend, owner common.Address) (addr common.Address, tx *types.Transaction, err error) {
-	addr, tx, s.Instance, err = contract.DeploySimpleSwap(auth, backend, owner)
+	addr, tx, s.Instance, err = contract.DeploySimpleSwap(auth, backend, owner, big.NewInt(0))
```

Comment:
Also if easy enough please rename the owner to issuer as agreed.

Reply:
Will do. Possibly there are more owners around

Reply:
Actually looking at it that should be done in a different PR. There is several instances in the rest of the Swap code.